apps/api/src/app/integrations/usecases/get-novu-integration/get-novu-integration.usecase.ts

```diff
@@ -54,7 +56,7 @@ export class GetNovuIntegration {
         providerId: CalculateLimitNovuIntegration.getProviderId(command.channelType),
         ...limit,
       });
-      throw new Error(`Limit for Novus ${command.channelType.toLowerCase()} provider was reached.`);
+      throw new ApiException(`Limit for Novus ${command.channelType.toLowerCase()} provider was reached.`);
```

Comment:
when the Novu provider limit is reached we should throw the `ApiException` not the general `Error`